lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -970,7 +970,7 @@ data UnsignedTx input = UnsignedTx
     { unsignedInputs
         :: NonEmpty input
     , unsignedOutputs
-        :: NonEmpty TxOut
+        :: [TxOut]
```

Comment:
Should add a comment here why we don't want NonEmpty. Will bight someone later, maybe.

Reply:
Good point, elaborated here: https://github.com/input-output-hk/cardano-wallet/compare/c60c5a60a6d1111e8dc5c7482a737bdc4e83c3b2..2bdbb29333beb82587055ab576619ec6760888f3